frontend/src/components/AboutDialog/AboutDialog.css

```diff
@@ -0,0 +1,10 @@
+#about-logo-div {
```

Comment:
in the rest of our css files we've used PascalCase for css classes. We also generally start with "." instead of #

Reply:
Ok, switched to using classes over IDs. Took a gander through the other stylesheets in the project and notice that there seems to only be a single instance of styling by ID. Is there an intent behind this @bcherny ? Like I'm not sure I understand why the img and div in this component need to be styled by class rather than by ID.

Reply:
It's generally bad practice to use IDs in CSS. It prevents reusing the style and having multiple instances of the component in the future.

Reply:
but @bcherny , how will i reference these elements using jQuery selectors!?

Reply:
What would your mom say if she heard that coming from you!